src/nft-card-back.ts

```diff
@@ -381,44 +416,44 @@ export class NftCardBackTemplate extends LitElement {
     `
   }
 
-  private formatTrait(trait) {
-    return trait.replace(/_/g, ' ')
-  }
-
-  private buildTraits(traitData) {
+  private buildTraits(traitData: TraitData) {
     this.traits = {
       props: [],
       stats: [],
       rankings: [],
       boosts: []
     }
-    const {traits, collectionTraits} = this.traitData
+    const {traits: assetTraits, collectionTraits} = traitData
 
-    for (const trait of traits) {
+    for (const trait of assetTraits) {
       const type = this.getTraitType(trait, collectionTraits)
 
       const name = trait.trait_type
+
       this.traits[type + 's'].push({
         name,
         value: trait.value,
-        ...(type === TraitType.Ranking ? { max: collectionTraits[name].max } : {})
+        ...(type === TraitType.Ranking ? { max: collectionTraits[name].max as unknown as number } : {}),
+        trait_type: '',
+        display_type: ''
```

Comment:
won't all display_types be blank?

Reply:
Yea I think I had that there to get rid of a TS error before I knew what I was doing 😆